### PR TITLE
replace old prepwork links and language

### PR DESCRIPTION
**Pull request by alvin-the-programmer**

### CODE REVIEW

File: _includes/technical-interview.md [hunk 1]
```diff
@@ -61,41 +52,14 @@ practice:
...
-
-- [Code Wars][code-wars]: A collection of coding challenges rated by difficulty. Once you solve a problem, you can see other programmers' solutions.
-
-- [Code Fights][code-fights]: Fun way to practice and test your programming skills against other programming players and bots. Let the fight begin.
-
-- [Project Euler][project-euler]: The first few challenges are fairly straightforward and can be brute-forced. As you get further into the challenges, you'll have to be more clever about optimizing your algorithms.
-
-- [LeetCode][leetcode]: A platform for preparing for technical coding interviews. The code you submit isn't only evaluated for correctness, but speed as well.
-
 Keep the following in mind when approaching problems:
```
Comment (wolverdude) [inline _includes/technical-interview.md]: Why keep this line? It's now meaningless.

File: _includes/technical-interview.md [hunk 2]
```diff
@@ -21,27 +21,18 @@ Watch these short videos to learn more:
...
 ## Technical Interview Prep Work
 
-Work through our [step 2 curriculum][step-two-curriculum]. This material is
+Work through our [Introduction to Programming I & II Courses][intro-to-programming]. These courses are
 designed to introduce concepts that may come in handy during your technical
-interview. Included are several practice assessments to give you opportunities
-to practice your new skills.
+interview. Included are many practice problems and video walkthroughs to give you opportunities to practice your new skills. When accessing the courses, be sure to have the correct course selected by clicking "Course Outline" in the top left corner and switching to "Introduction to Programming" if you do not already have it selected.
 
 Read our [introduction to pair programming][pair-programming]. Your interview
```
Comment (wolverdude) [inline _includes/technical-interview.md:28]: Do we have updated material on pair programming, or is this link still current?

**Review by wolverdude — CHANGES_REQUESTED** [review]

File: _includes/technical-interview.md (hunk 2 above)
Comment (alvin-the-programmer) [reply _includes/technical-interview.md:28]: That is current, there is no other version of this.

File: _includes/technical-interview.md (hunk 1 above)
Comment (alvin-the-programmer) [reply _includes/technical-interview.md]: removed

**Comment by wolverdude** [comment]
Thanks!

**Pull request merged**